config/database.example.yml

```diff
@@ -5,16 +5,17 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: mysql2
-  encoding: utf8
+  adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
-  password: root
-  socket: /tmp/mysql.sock
+  username: postgres
+  password: postgres
+
 
 development:
   <<: *default
   database: circuitverse_development
+  host: localhost
```

Comment:
test environment will also have `host: localhost`, maybe this can be included in `default` and mentioned in both development and test sections. Production can have its own section.